app/models/grade_entry_student.rb

```diff
@@ -204,4 +205,10 @@ def refresh_total_grade
 
     self.total_grade = total
   end
+
+  def update_results
+    grade_entry_form = GradeEntryForm.find(self.assessment_id)
```

Comment:
There's a better way to access the associated grade entry form (note the `belongs_to` association)